pull req updated features

gyanplores opened the pull request:
post authors, comment authors, and reply authors now have visible photos in the posts
can upvote comments and replies

Shinoruba committed:
Merge pull request #1 from user1angelo/backend-feature

[REVAMP]

Shinoruba committed:
Merge pull request #2 from user1angelo/backend-feature

forgot to change something

Shinoruba committed:
Merge pull request #3 from user1angelo/backend-feature

[MAJOR] Forum Page and Admin Things

Shinoruba committed:
[Minor]
1. Announcements in index page

gyanplores committed:
updated changes

gyanplores committed:
atlas update

gyanplores committed:
UPDATED

gyanplores committed:
Updated Features

Shinoruba reviewed (commented):
This is good, I have a suggestion though, can we remove the search option in the header if we are not going to implement the functionality for it?

gyanplores commented:
i dont think we have enough products to have the need for a search bar in my opinion lmao. But i think we should get rid of the cart kase the idea was to redirect them to the facebook page right?

Shinoruba committed:
[Fixed Bugs]:
1. Editing product in the admin dashboard now works.
2. 2 products cannot share the same int 'code'.
3. Changed button in forum post from "Edit Comment" to "Edit Post".

Shinoruba committed:
commenting out
